lib/auth/init_test.go

```diff
@@ -906,6 +906,20 @@ func TestPresets(t *testing.T) {
 	})
 }
 
+func TestGetPresetUsers(t *testing.T) {
+	// len is 0 for OSS
+	modules.SetTestModules(t, &modules.TestModules{
+		TestBuildType: modules.BuildOSS,
+	})
+	require.Empty(t, len(getPresetUsers()))
+
+	// len is 1 for enterprise (@teleport-access-approval-bot)
+	modules.SetTestModules(t, &modules.TestModules{
+		TestBuildType: modules.BuildEnterprise,
+	})
+	require.Len(t, getPresetUsers(), 1)
```

Comment:
Should we check that this contains exactly the result of NewSystemAutomaticAccessBotUser, in both cases (OSS and Enterprise modules)?

Reply:
For OSS we have to check for an empty slice, since `NewSystemAutomaticAccessBotUser` returns `nil`:

```go
// oss
require.Equal(t, []types.User{}, getPresetUsers())

// enterprise
require.Equal(t, []types.User{
	services.NewSystemAutomaticAccessBotUser(),
}, getPresetUsers())
```

What do you think? I like this better than the current len check.

Reply:
I would do require.Empty for the first, but otherwise seems OK.